politeiawww/api/cms/v1/v1.go

```diff
@@ -127,13 +129,13 @@ type InvoiceDetailsReply struct {
 // InvoiceInput is the expected structure of the invoice.json file being added to InvoiceRecords.
 // Users' raw csv will be inputted and parsed to help in their creation.
 type InvoiceInput struct {
-	ID                 string           `json:"id"`                 // Optional field for contractor ID entry
+	Version            uint16           `json:"version"`            // Version of the invoice input
```

Comment:
Just use uint or int.

The JSON that hits the wire will encode this as a string so from a size perspective it doesn't matter. On the machine language side all uint16s will be padded to 64 bit for various performance and access reasons. There are no savings in using a uint16 in this case. In addition, using odd types also results in a bunch of unecessary typecasting throughout the code.